src/extension.ts

```diff
@@ -634,56 +640,45 @@ function launchMetals(
         );
       });
     });
-    if (features.treeViewProvider) {
```

Comment:
So I see that there are a few checks here to ensure that these features are on, but AFIK, these are sort of pointless right? Since we are the ones setting them, and we always have them, these should be fine to remove right?

Reply:
Should be fine I think. The only issue might be if someone want's to go waaaay back.